SocialFilter/sections/specific.txt

```diff
@@ -4545,3 +4545,8 @@ thetalko.com##.btn-facebook-like
 barmer.de##.sidebar > .siteactions
 !+ PLATFORM(ios, ext_android_cb)
 bildderfrau.de##.article--sticky
+!+ PLATFORM(ios, ext_android_cb)
```

Comment:
The existing rules (like `~vk.com,~vkontakte.ru##a[href^="http://vkontakte.ru/share.php"]`)
 do not work in Samsung browser